home.html

```diff
@@ -0,0 +1,239 @@
+          <p class="program-detail">
+            Appreciate var  ious creations applying CC license of artists,
+            organized from Art Center Nabi
+          </p>
+        </div>
+        <div class="program">
+          <img src="./images/program_icon/program_icon_03.png" alt="" />
+          <p class="program-title">Forum</p>
+          <p class="program-detail">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+        <div class="program">
+          <img src="./images/program_icon/program_icon_04.png" alt="" />
+          <p class="program-title">Workshop</p>
+          <p class="program-detail">
+            Try creating your own work using open source license rather than
+            just watching at it.
+          </p>
+        </div>
+        <div class="program">
+          <img src="./images/program_icon/program_icon_05.png" alt="" />
+          <p class="program-title">CC Ignite</p>
+          <p class="program-detail">
+            Get opportunities to network with CC affiliates around the world,
+            also after the summit.
+          </p>
+        </div>
+      </div>
+      <button class="join-button" type="button">Join CC Global Summit 2015</button>
+      <button class="see-button" type="button">SEE THE WHOLE PROGRAM</button>
+    </section>
+
+    <!-- Speaker Section -->
+
+    <section class="speaker-section">
+      <h3 class="speaker-section-title">Featured Speakers
+        <hr class="underline"></hr>
+      </h3>
+      <article class="speaker-list" id="speaker-list">
+        <!-- <div class="speaker">
+          <div class="img-container">
+              <img src="./images/speakers/speaker_01.png" alt="speaker">
+          </div>
+          <div class="speaker-info-container">
+            <h4 class="speaker-name">SohYeong Noh</h4>
+            <p class="speaker-summary">Director of Art Centre Nabi and a board member of CC Korea</p>
+            <hr class="speaker-hr">
+            <p class="speaker-studies">As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding.</p>
+          </div>
+        </div>
+        <div class="speaker">
+          <div class="img-container">
+              <img src="./images/speakers/speaker_01.png" alt="speaker">
+          </div>
+          <div class="speaker-info-container">
+            <h4 class="speaker-name">SohYeong Noh</h4>
+            <p class="speaker-summary">Director of Art Centre Nabi and a board member of CC Korea</p>
+            <hr class="speaker-hr">
+            <p class="speaker-studies">As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding.</p>
+          </div>
+        </div>
+        <div class="speaker">
+          <div class="img-container">
+              <img src="./images/speakers/speaker_01.png" alt="speaker">
```

Comment:
- [ ] Please kindly remove all forms of commented code from your codebase as it pollutes it.

Reply:
Done